setup.cfg

```diff
@@ -46,9 +46,8 @@ asdf_extensions =
 
 [options.extras_require]
 test =
-    pytest-astropy
+    pytest-astropy>=0.8
     pytest-xdist
-    pytest-mpl
```

Comment:
`pytest-mpl` is not part of `pytest-astropy`, so should we remove it from here?

Reply:
I think the reason I didn't include it for now is because if we leave it here it means we can never actually run the test suite without matplotlib - I need to go and make pytest-mpl not require matplotlib before we can add it back (at which point we'll just add it to pytest-astropy).

Reply:
Okay, I re-opened astropy/pytest-astropy#4 so we can follow-up there.